packages/connect/package.json

```
@@ -39,7 +39,7 @@
     "eventemitter3": "^4.0.7",
     "file-entry-cache": "^6.0.1",
     "mkdirp": "^1.0.4",
-    "smoldot": "0.2.9"
+    "smoldot": "0.3.1"
```

```suggestion
    "smoldot": "0.3.2"
```

Just published 0.3.2